js/src/wp-seo-replacevar-plugin.js

```diff
@@ -529,5 +529,14 @@ var ReplaceVar = require( "./values/replaceVar" );
 		return parentText;
 	};
 
+	/**
+	 * STATIC VARIABLES
+	 */
+
+	/**
```

Comment:
I've kept it consistent with the other sections already used in the ReplaceVar Plugin. But I see the Shortcode Plugin uses a different style, I'll update the PR to keep the ReplaceVar Plugin consistent with the Shortcode Plugin.